backend/api/helper/permissions.py

```diff
@@ -0,0 +1,16 @@
+from flask_security import current_user
+
+
+def current_user_has_permission_to_change(entry) -> bool:
```

Comment:
Also if we already have a permission help class, we should also put the global ADMIN_PERMISSION in here. (or well, delete it since we don't use it currently)